Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -128,7 +128,9 @@ enum Pattern: String {
     case uiFailingTest = #"^\s{4}t = \s+\d+\.\d+s\s+Assertion Failure: (.*:\d+): (.*)$"#
 
     /// Regular expression captured groups:
-    case restartingTests = #"^Restarting after unexpected exit.+$"#
+    /// $3 = test suite
+    /// $4 = test case
+    case restartingTest = #"^Restarting after unexpected exit, crash, or test timeout in ((-\[(\w+)\s(\w+)\])|((\w+)\.(\w+)\(\)));.*"#
```

Comment:
Can you also list other groups?

Reply:
I removed one that wasn't useful and updated the code + doc